src/Traits/LogsActivity.php

```diff
@@ -46,6 +51,16 @@ protected static function bootLogsActivity()
         });
     }
 
+    public function getSubmitEmptyLogs (): bool
+    {
+        return is_null($this->submitEmptyLogs) ? true : $this->submitEmptyLogs;
+    }
+
+    public function logIsEmpty ($attrs): bool
```

Comment:
`isLogEmpty()` - methods with a Boolean return should start with `has|is|should`